integration/fuse/pom.xml

```diff
@@ -63,7 +63,39 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-
+  <profiles>
+    <profile>
+      <id>native</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <properties>
+        <runningWithNative>true</runningWithNative>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>native_fuse_compile</id>
+                <phase>compile</phase>
+                <goals><goal>run</goal></goals>
+                <configuration>
+                  <target>
+                    <mkdir dir="${project.build.directory}/native"/>
+                    <exec executable="make" failonerror="true" dir="${project.build.directory}/native">
```

Comment:
it seems I have to change this to `<exec executable="make" failonerror="true"
                          dir="${project.basedir}/src/main/native/libjnifuse"/>`
or the build will fail

Reply:
@apc999 Maybe something wrong here, but, I aimed to change the working directory from the source directory to a generated directory `${project.build.directory}/native"`. Let me check why it failed.

Reply:
I forget to add a `-f` argument to specify the makefile. It's my bad.